### ISSUE TITLE
A11y: Announce Filter settings on load, and when done

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/clean-and-green-philly#341]
## Description

When a screenreader user navigates to Filter settings, we don't announce the change of context. We also don't announce the change after filters has been set. Dynamic changes in the page should be announced so that screen reader users are notified.

## Recommendation
On Filter load
- We can use `aria-live` region, then programmatically put the focus on the H2 `Filter` on load
- Or just programmatically put the focus on the H2 `Filter` on load.

On Filter - done
- Use `aria-live` region and announce "results updated"
- Programmatically manage the focus on the results list.

**Comment by marvieqa** [comment]
We should first implement the filter design/workflow changes described in this [slack thread](https://codeforphilly.slack.com/archives/C05H9QBMP96/p1711032217692499), and then determine whether this ticket is still relevant or can be closed.

**Comment by marvieqa** [comment]
This is no longer relevant since we've changed the design. Closing issue.
